test/sendReceive.spec.ts

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import "mocha";
+import * as chai from "chai";
+const should = chai.should();
+import * as chaiAsPromised from "chai-as-promised";
+import * as dotenv from "dotenv";
+dotenv.config();
+chai.use(chaiAsPromised);
+import {
+  Namespace,
+  QueueClient,
+  SendableMessageInfo,
+  generateUuid,
+  ServiceBusMessage,
+  TopicClient,
+  SubscriptionClient
+} from "../lib";
+
+describe("Simple send/receive to/from Queue/Topic/Subscription", function() {
+  let namespace: Namespace;
+  let queueClient: QueueClient;
+  let topicClient: TopicClient;
+  let subscriptionClient: SubscriptionClient;
+
+  beforeEach(async () => {
```

Comment:
Is there anything special about the queues? As they need to be created prior to the tests running if they need any special setup we should capture it here.

Reply:
Note added